src/cpp/session/modules/rmarkdown/NotebookOutput.cpp

```diff
@@ -170,6 +178,24 @@ Error fillOutputObject(const std::string& docId, const std::string& chunkId,
 
       (*pObj)[kChunkOutputValue] = url;
    }
+   else if (outputType == ChunkOutputData)
+   {
+      SEXP argsSEXP;
+      r::sexp::Protect rProtect;
+      Error error = r::exec::RFunction(".rs.readDataCapture", path.absolutePath()).call(
```

Comment:
We need to do the same utf8toSystem path conversion Kevin mentioned previously here.